<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1838</width>
    <height>985</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="cargar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>840</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="examinar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>790</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Examinar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="file">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>800</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>780</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre de archivo: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>830</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre del paciente:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>850</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>780</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID paciente:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_paciente">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>800</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>830</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha de nacimiento:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nacimiento">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>850</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>780</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sexo:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sex">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>800</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>830</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID estudio:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="estudio">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>850</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>780</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha de estudio:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="date_estudio">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>800</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>830</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo de estudio:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="time_estudio">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>850</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>780</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre médico:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="medico">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>800</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>830</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero de serie: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serie">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>850</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt; color:#00557f;&quot;&gt;CARGA DE IMÁGENES DICOM&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="graficar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>890</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Graficar ejes</string>
    </property>
   </widget>
   <widget class="ImageView" name="gra">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>471</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="ImageView" name="gra_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>100</y>
      <width>471</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="ImageView" name="gra_3">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>100</y>
      <width>471</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="h_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>520</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Histograma</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="h_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>520</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Histograma</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="h_3">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>520</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Histograma</string>
    </property>
   </widget>
   <widget class="ImageView" name="gra_4">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>600</y>
      <width>331</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_anterior">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>880</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_siguiente">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>880</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>660</y>
      <width>561</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; color:#00557f;&quot;&gt;Gráfico DICOM&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; color:#00557f;&quot;&gt;Gráfico ejes DICOM&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1851</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>580</y>
      <width>1891</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>910</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mover archivo anterior </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>910</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mover archivo siguiente </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_anterior_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>480</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_siguiente_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_siguiente_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_anterior_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>480</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_siguiente_4">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>100</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_anterior_4">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>480</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rot_m1">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>rotar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rot_m2">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>rotar </string>
    </property>
   </widget>
   <widget class="QPushButton" name="rot_m3">
    <property name="geometry">
     <rect>
      <x>1610</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>rotar</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>31</width>
      <height>351</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_2">
    <property name="geometry">
     <rect>
      <x>631</x>
      <y>130</y>
      <width>31</width>
      <height>351</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_3">
    <property name="geometry">
     <rect>
      <x>1201</x>
      <y>130</y>
      <width>31</width>
      <height>351</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1838</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCargar_Dicom">
    <property name="title">
     <string>Cargar Dicom</string>
    </property>
   </widget>
   <addaction name="menuCargar_Dicom"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
